src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -120,24 +120,24 @@ public String getApiUri() {
     @Override public void visitSources(SCMSourceObserver observer) throws IOException, InterruptedException {
         TaskListener listener = observer.getListener();
         if (repoOwner.isEmpty()) {
-            listener.getLogger().format("Must specify user or organization%n");
-            return;
+            throw new AbortException("Must specify user or organization%n");
```

Comment:
> Owner would be considered a human error

Which `FormValidation` should already be catching.

The problem here is that `repoOwner` will be blank for a newly created folder—a normal condition.